Lib/pickle.py

```diff
@@ -1909,20 +1909,17 @@ def _loads(s, /, *, fix_imports=True, encoding="ASCII", errors="strict",
 
 if __name__ == "__main__":
     import argparse
+    import pprint
     parser = argparse.ArgumentParser(
         description='display contents of the pickle files')
     parser.add_argument(
         'pickle_file',
-        nargs='*', help='the pickle file')
+        nargs='+', help='the pickle file')
     args = parser.parse_args()
-    if not args.pickle_file:
-        parser.print_help()
-    else:
-        import pprint
-        for fn in args.pickle_file:
-            if fn == '-':
-                obj = load(sys.stdin.buffer)
-            else:
-                with open(fn, 'rb') as f:
-                    obj = load(f)
-            pprint.pprint(obj)
+    for fn in args.pickle_file:
+        if fn == '-':
+            obj = load(sys.stdin.buffer)
+        else:
+            with open(fn, 'rb') as f:
+                obj = load(f)
+        pprint.pprint(obj)
```

Comment:
Shall we prefer `pp`? It defaults `sort_dicts=False` and often has better output (we moved it to the top of the docs to try and prioritise it):

https://docs.python.org/3/library/pprint.html

```suggestion
        pprint.pp(obj)
```

Reply:
I like this idea :+1:

Reply:
Personally, I think `sort_dicts=True` is better because it's easier to find what you're looking for. However, if you know what you're pickling, then maybe it's easier to locate it as well. If we made it top-level in the docs, maybe let's use it here as well? (since dicts are sorted by insertion time, maybe it's better to render correct insertion time as well)

Reply:
I back sorting

Reply:
Let's either do `pprint.pp(obj)` or `pprint.pprint(obj)`

I prefer `pp` but won't block if @picnixz prefers `pprint`.

Reply:
No I'm ok with pp